src/open_inwoner/accounts/tests/test_profile_views.py

```diff
@@ -346,6 +350,86 @@ def test_expected_form_is_rendered(self):
 
         self.assertEqual(type(form), BrpUserForm)
 
+    def test_image_is_saved_when_begeleider_and_default_login(self):
+        self.user.contact_type = ContactTypeChoices.begeleider
+        self.user.save()
+
+        image = Image.new("RGB", (10, 10))
+        byteIO = io.BytesIO()
+        image.save(byteIO, format="png")
+        img_bytes = byteIO.getvalue()
```

Comment:
If you have repeating blocks of setup or utility code in the tests it is helpful to move it to a method or function so we have less uninteresting/noisy/distracting stuff in the tests.